.github/workflows/ci.yml

```diff
@@ -86,4 +86,29 @@ jobs:
         with:
           go-version-file: go.mod
           cache: false
+
+      - uses: elastic/apm-pipeline-library/.github/actions/docker-login@current
+        with:
+          registry: docker.elastic.co
+          secret: secret/observability-team/ci/docker-registry/prod
+          url: ${{ secrets.VAULT_ADDR }}
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+        if: |
+          github.event_name != 'pull_request'
+          || github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false
+
+      - run: echo "GENERATE_WOLFI_IMAGES=true" >> "$GITHUB_ENV"
```

Comment:
https://github.com/elastic/apm-server/pull/13138 is the one validating these conditionals in a forked repo